tests/FSharp.Compiler.ComponentTests/Language/ObsoleteAttributeCheckingTests.fs

```diff
@@ -237,9 +236,30 @@ type Color =
     
 let c = Color.Red
         """
-        |> ignoreWarnings
         |> compile
-        |> shouldSucceed
+        |> shouldFail
+        |> withDiagnostics [
+            (Error 101, Line 5, Col 7, Line 5, Col 50, "This construct is deprecated. Use B instead")
+            // FIXME Find the reason why we are getting this new compiler error
+            (Error 39, Line 8, Col 15, Line 8, Col 18, "The type 'Color' does not define the field, constructor or member 'Red'.")
```

Comment:
@dsyme  I'm trying to figure why this second error is  raised after check for obsolete attributes on Enums.  Any pointers that could help ?

Reply:
> I think this is ready to go, @edgarfgp fine with converting from draft to open?

I'm just trying to figure why I'm getting after my change
```fsharp

 (Error 39, Line 8, Col 15, Line 8, Col 18, "The type 'Color' does not define the field, constructor or member 'Red'.")
```

Reply:
Also the range for the first error, right now saying line 5 (= definition) should rather be line 8 (=usage).
Something is fishy when it comes tu IsError=true usage of obsolete,...

Reply:
@T-Gro Im kind  stuck at the moment .I will give a fresh look over the weekend :)

Reply:
@T-Gro This is now fixed . Turns out the fix was more simple 😀